solr/core/src/java/org/apache/solr/cloud/ZkController.java

```diff
@@ -653,7 +653,12 @@ public void close() {
     customThreadPool.submit(() -> Collections.singleton(overseer).parallelStream().forEach(IOUtils::closeQuietly));
```

Comment:
While we're here, this line (and a few others) should be `customThreadPool.submit(() -> IOUtils.closeQuietly(overseer);` I have no idea why we're creating a collection and a stream for a single object.

Reply:
I have cleaned it up